broccoli/addon.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+/* eslint-env node */
+
+const Funnel = require('broccoli-funnel');
+
+module.exports = function addon(name, project) {
+  var addon = project.findAddonByName(name);
+  var tree = addon.treeFor('addon');
+
+  return new Funnel(tree, {
+    srcDir: 'modules'
```

Comment:
This changes in ember-cli@2.13.  We no longer emit `treeForAddon` into `modules/`.